src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -578,6 +578,24 @@ protected override ExpressionSyntax CreateCallSignature()
                     return invocation;
                 }
 
+                if (this.CSharpSelectionResult.ShouldCallConfigureAwaitFalse())
+                {
+                    if (this.AnalyzerResult.ReturnType.GetMembers().Any(x => x is IMethodSymbol
+                    {
+                        Name: nameof(Task.ConfigureAwait),
+                        Parameters: { Length: 1 } parameters
+                    } && parameters[0].Type is { SpecialType: SpecialType.System_Boolean }))
```

Comment:
this doesn't feel like it needs a pattern.  just do `.Type.SpecialType == Boolean`.